docs/index.md

```diff
@@ -0,0 +1,40 @@
+Container Object Storage Interface (COSI)
+------------------------------------------
+
+Container Object Storage Interface (COSI) is a set of abstractions for provisioning and management of object storage. It aims to be a common layer of abstraction across multiple object storage vendors, such that workloads can request and automatically be provisioned object storage buckets. 
+
+The goals of this project are:
+
+ - Automate object storage provisioning, access and management
+ - Provide a common layer of abstraction for consuming object storage
+ - Facilitate lift and shift of workloads across object storage providers (i.e. prevent vendor lock-in)
```

Comment:
Instead of "lift and shift", or in addition to this goal I would include the important goal of app portability across the same protocols.

Reply:
do you have a one-liner that i could just add to that list?